StratisCore.UI/src/app/wallet/tokens/components/add-token/add-token.component.ts

```diff
@@ -46,7 +57,8 @@ export class AddTokenComponent implements OnInit, OnDestroy, Disposable {
   ngOnInit() {
   }
 
-  ngOnDestroy() {
+  public ngOnDestroy() {
+    this.subscriptions.forEach(sub => sub.unsubscribe());
```

Comment:
Wouldn't need forEach here, would just be `this.subscriptions.unsubscribe();`

Reply:
As above